content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -2,127 +2,143 @@
-section, and select :guilabel:`Received quantities`. Then, :guilabel:`Save` changes.
-
-In the :guilabel:`Purchase` app, create a new :abbr:`RFQ (Request for Quotation)`. Fill out the
-information on the quotation form, add products to the invoice lines, and click :guilabel:`Confirm
-Order`. Then, click on the :guilabel:`Receipt smart button`. Set the quantities in the
-:guilabel:`Done` column to match the quantities in the :guilabel:`Demand` column, and
-:guilabel:`Validate` the changes. Then, in the purchase order, click :guilabel:`Create Bill` and
-:guilabel:`Confirm`. Since the policy is set to *received quantities*, the draft bill can be
-confirmed *only* when at least some of the quantities are received.
+   Navigate to :menuselection:`Purchase app --> Products --> Products`, and select a product. From
+   the product form, click the :guilabel:`Purchase` tab. Under the :guilabel:`Vendor Bills` section,
+   modify the selection in the :guilabel:`Control Policy` field, and :guilabel:`Save` changes.
 
 3-way matching
 ==============
 
-Activating :guilabel:`3-way matching` ensures that vendor bills are only paid once some or all of
-the products included in the purchase order have actually been received. To activate it, go to
-:menuselection:`Purchase --> Configuration --> Settings`, and scroll down to the
-:guilabel:`Invoicing` section. Then, click :guilabel:`3-way matching: purchases, receptions, and
-bills`.
+The *3-way matching* feature ensures vendor bills are only paid once some or all of the products
+included in the |PO| have been received.
 
-.. note::
-   3-way matching is *only* intended to work when the bill control policy is set to *received
+To activate *3-way matching*, navigate to :menuselection:`Purchase app --> Configuration -->
+Settings`, and scroll down to the :guilabel:`Invoicing` section. Then, click the checkbox for
+:guilabel:`3-way matching` to enable the feature.
+
+.. image:: control_bills/control-bills-three-way-matching.png
+   :align: center
+   :alt: Enabled 3-way matching feature in Purchase app settings.
+
+.. important::
+   The *3-way matching* feature **only** works with the *bill control* policy set to *Received
    quantities*.
 
 Pay vendor bills with 3-way matching
 ------------------------------------
 
-When :guilabel:`3-way matching` is activated, vendor bills will display the :guilabel:`Should Be
-Paid` field under the :guilabel:`Other Info` tab. When a new vendor bill is created, the field will
-be set to :guilabel:`Yes`, since a bill can't be created until at least some of the products
-included in a purchase order have been received.
+When *3-way matching* is enabled, vendor bills display a :guilabel:`Should Be Paid` field under the
+:guilabel:`Other Info` tab. When a new vendor bill is created, the field is set to :guilabel:`Yes`,
+since a bill **can't** be created until at least some of the products included in a |PO| have been
+received.
+
```

Comment:
Can you please describe how the user can find/locate a vendor bill? That sort of elaboration would be beneficial at this point in the doc

Reply:
The paragraph directly below this one explains how to create a vendor bill, I thought was a good explanation/way to structure it. If not, let me know by confirming that it's not again and I will add an additional paragraph or note to explain where to find it